src/lib/y2packager/dialogs/inst_product_license.rb

```diff
@@ -55,11 +55,24 @@ def contents
         VBox(
           Widgets::ProductLicenseTranslations.new(product, Yast::Language.language),
           HBox(
-            Left(Widgets::ProductLicenseConfirmation.new(product)),
+            confirmation_checkbox,
```

Comment:
how it works previously for opensuse which does not have license or for some addons?

Reply:
That's a good question.

Maybe because (the installer workflow?) is using https://github.com/yast/yast-packager/blob/master/src/lib/y2packager/widgets/product_license.rb instead?

What do you think?

Reply:
so different dialog. It is possible.